packages/warriorjs-cli/src/Profile.test.js

```diff
@@ -38,6 +39,15 @@ describe('Profile.load', () => {
     const profile = await Profile.load('/path/to/profile');
     expect(profile).toBeNull();
   });
+
+  test('returns null if on an invalid profile directory', async () => {
+    Profile.read = () =>
+      'eyJ3YXJyaW9yTmFtZSI6ICJKb2UiLCAidG93ZXJOYW1lIjogImJlZ2lubmVyIiwgImZvbyI6IDQyfQ==';
+    Profile.isProfileDirectory = () => false;
+
+    const profile = await Profile.load('/path/to/profile');
+    expect(profile).toBeNull();
+  });
 });
 
```

Comment:
It would be nice if we had tests for `Profile.isProfileDirectory` that leverage `mock-fs`. We should test the result of that method when both files exist, only one of them exists and none exist.